### PR TITLE
Handle errors

### PR DESCRIPTION
**Pull request by keenlok**

### CODE REVIEW

**Comment by YuPeiHenry** [comment]
I'm curious, how did ShortWindow.html end up failing the check? I thought it was generated by processResources. Might be my fault somehow.

**Comment by keenlok** [comment]
Not sure, it says it needs new line at EOF, but cant seem to get it to work

File: docs/AboutUs.adoc [hunk 1]
```diff
@@ -40,7 +40,7 @@ Responsibilities: Storage
 
 === Leong Engsea
 image::leongengsea.png[width="150", align="left"]
-{empty}[https://github.com/leongengsea[github]] [<<johndoe#, portfolio>>]
+{empty}[https://github.com/leongengsea[github]] [<<leongengsea#, portfolio>>]
```
Comment (YuPeiHenry) [inline docs/AboutUs.adoc]: 👍 Thanks for helping Jia Jun and Engsea to correct the wrong portfolio links.

File: docs/ShortHelp.adoc [hunk 2]
```diff
@@ -7,7 +7,7 @@
 * `delete <index>`
 * `exit`
 * `filter <tag> <condition>`
-* `find <attraction>` or `find <tag> <address>`
+* `find <attraction>` or `find <tag> <zone>`
```
Comment (YuPeiHenry) [inline docs/ShortHelp.adoc]: Good idea.

File: docs/team/johndoe.adoc [hunk 3]
```diff
@@ -9,7 +9,7 @@
 
 == Overview
 
-AddressBook - Level 4 is a desktop address book application used for teaching Software Engineering principles. The user interacts with it using a CLI, and it has a GUI created with JavaFX. It is written in Java, and has about 10 kLoC.
+AddressBook - Level 4 is a desktop zone book application used for teaching Software Engineering principles. The user interacts with it using a CLI, and it has a GUI created with JavaFX. It is written in Java, and has about 10 kLoC.
```
Comment (YuPeiHenry) [inline docs/team/johndoe.adoc]: Hmm, did you replace all instances of address? No harm here though.

File: src/test/java/seedu/thanepark/model/ride/MaintenanceTest.java [hunk 4]
```diff
@@ -47,17 +47,17 @@ public void isValidMaintenance() {
...
-        // invalid phone numbers
+        // invalid numbers
         assertFalse(Maintenance.isValidMaintenance("")); // empty string
         assertFalse(Maintenance.isValidMaintenance(" ")); // spaces only
         assertFalse(Maintenance.isValidMaintenance("phone")); // non-numeric
         assertFalse(Maintenance.isValidMaintenance("9011p041")); // alphabets within digits
         assertFalse(Maintenance.isValidMaintenance("9312 1534")); // spaces within digits
+        assertFalse(Maintenance.isValidMaintenance("124293842033123")); // long phone numbers
 
-        // valid phone numbers
```
Comment (YuPeiHenry) [inline src/test/java/seedu/thanepark/model/ride/MaintenanceTest.java]: Thanks for removing "phone".

File: docs/team/johndoe.adoc (hunk 3 above)
Comment (keenlok) [reply docs/team/johndoe.adoc]: oops didnt see that

**Pull request merged**